service/service.go

```diff
@@ -142,8 +143,14 @@ func New(config Config) (*Service, error) {
 				Region:          config.Viper.GetString(config.Flag.Service.AWS.Region),
 			},
 			InstallationName: config.Viper.GetString(config.Flag.Service.Installation.Name),
-			Name:             config.Name,
-			PubKeyFile:       config.Viper.GetString(config.Flag.Service.AWS.PubKeyFile),
+			K8sAPIExtraArgs: []string{
```

Comment:
... I mean here `c := awsconfig.FrameworkConfig{` <- above this line you can construct extra args only if the flag values aren't empty.